Observer/AdminhtmlCustomerSaveAfterObserver.php

```diff
@@ -0,0 +1,834 @@
+     * @var CollectionFactory
+     */
+    protected $_collectionFactory;
+
+    /**
+     * @var \Magento\Framework\Stdlib\DateTime\DateTime
+     */
+    protected $_date;
+
+    /**
+     * Store manager.
+     *
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var \Magento\Catalog\Api\ProductRepositoryInterface
+     */
+    protected $_productRepository;
+
+    /**
+     * @var \Magento\Framework\Message\ManagerInterface
+     */
+    private $_messageManager;
+
+    protected $_mediaDirectory;
+
+    /**
+     * @var \Webkul\Marketplace\Model\ResourceModel\Product\Collection
+     */
+    protected $_sellerProduct;
+
+    /**
+     * @var \Magento\Framework\Json\DecoderInterface
+     */
+    protected $_jsonDecoder;
+
+    /**
+     * @var MpSalesPartner
+     */
+    protected $mpSalesPartner;
+
+    /**
+     * @var MpSeller
+     */
+    protected $mpSeller;
+
+    /**
+     * @var \Magento\Customer\Model\CustomerFactory
+     */
+    protected $customerFactory;
+
+    /**
+     * @var MpHelper
+     */
+    protected $mpHelper;
+
+    /**
+     * @var MpEmailHelper
+     */
+    protected $mpEmailHelper;
+
+    /**
+     * @var \Webkul\Marketplace\Model\ProductFactory
+     */
+    protected $mpProductFactory;
+
+    /**
+     * @var \Magento\Framework\Module\Dir\Reader
+     */
+    protected $reader;
+
+    /**
+     * @var Filesystem
+     */
+    protected $filesystem;
+
+    /**
+     * @var ProductModel
+     */
+    protected $productModel;
+
+    /**
+     * @param Filesystem                                       $filesystem,
+     * @param \Magento\Framework\Stdlib\DateTime\DateTime      $date,
+     * @param \Magento\Framework\Message\ManagerInterface      $messageManager,
+     * @param \Magento\Store\Model\StoreManagerInterface       $storeManager,
+     * @param \Magento\Catalog\Api\ProductRepositoryInterface  $productRepository,
+     * @param CollectionFactory                                $collectionFactory,
+     * @param \Magento\MediaStorage\Model\File\UploaderFactory $fileUploaderFactory
+     * @param ProductCollection                                $sellerProduct
+     * @param \Magento\Framework\Json\DecoderInterface         $jsonDecoder
+     * @param MpSalesPartner                                   $mpSalesPartner
+     * @param MpSeller                                         $mpSeller
+     * @param \Magento\Customer\Model\CustomerFactory          $customerFactory
+     * @param MpHelper                                         $mpHelper
+     * @param MpEmailHelper                                    $mpEmailHelper
+     * @param \Webkul\Marketplace\Model\ProductFactory         $mpProductFactory
+     * @param \Magento\Framework\Module\Dir\Reader             $reader
+     * @param Filesystem                   $filesystem
+     * @param ProductModel                                     $productModel
+     */
+    public function __construct(
```

Comment:
import (use) namespaces at the beginning of the file

Reply:
Yes, will setup magento-coding-standard scan and will fix